rbac/management/access/view.py

```diff
@@ -103,23 +104,33 @@ def get_queryset(self, ordering):
     def get(self, request):
         """Provide access data for principal."""
         # Parameter extraction
-        sub_key, ordering = self.validate_and_get_param(request.query_params)
-
-        principal = get_principal_from_request(request)
-        if settings.AUTHENTICATE_WITH_ORG_ID:
-            cache = AccessCache(request.tenant.org_id)
-        else:
-            cache = AccessCache(request.tenant.tenant_name)
-        access_policy = cache.get_policy(principal.uuid, sub_key)
-        if access_policy is None:
-            queryset = self.get_queryset(ordering)
-            access_policy = self.serializer_class(queryset, many=True).data
-            cache.save_policy(principal.uuid, sub_key, access_policy)
-
-        page = self.paginate_queryset(access_policy)
-        if page is not None:
-            return self.get_paginated_response(page)
-        return Response({"data": access_policy})
+
+        tracer = trace.get_tracer(__name__)
+        with tracer.start_as_current_span("access-get"):
+            span = trace.get_current_span()
```

Comment:
Is it possible to get the span from the context manager in the `with` clause instead of calling `span = trace.get_current_span()`?  Something like this:

`
   with tracer.start_as_current_span("access-get") as span:
`